roles/mongodb_auth/molecule/default/tests/test_default.py

```diff
@@ -44,7 +44,7 @@ def test_mongo_shell_connectivity(host):
     facts = include_vars(host)["ansible_facts"]
     port = facts.get("mongod_port", 27017)
     user = facts.get("mongod_admin_user", "admin")
-    pwd = facts.get("mongod_default_admin_pwd", "admin")
+    pwd = facts.get("mongod_admin_pwd", "adminchangedpassword")
```

Comment:
Note the variable name here. This lookup would have always failed and the default value would have been provided. THis line should be
`pwd = facts.get("mongodb_admin_pwd", "admin")`